src/utils/sample.rs

```diff
@@ -7,16 +7,16 @@ use std::cmp::min;
 pub(crate) fn sample_uniform(hash: &[u8], n: u64) -> Option<u64> {
     // Computes the integer reprensation of hash* modulo n when n is not a
     // power of two. *(up to 8 bytes, in little endian)
-    fn mod_non_power_of_2(hash: &[u8], n: u64) -> Option<u64> {
+    fn mod_noset_sizeower_of_2(hash: &[u8], n: u64) -> Option<u64> {
```

Comment:
confused with the function renaming

Reply:
This is a oversight of mine when changing all `n_p` to `set_size`.

Reply:
+1